articles/container-apps/networking.md

```diff
@@ -401,6 +401,9 @@ The static IP address of the Container Apps environment is available in the Azur
 
 When you deploy an internal or an external environment into your own network, a new resource group is created in the Azure subscription where your environment is hosted. This resource group contains infrastructure components managed by the Azure Container Apps platform. Don't modify the services in this group or the resource group itself.
 
+> [!NOTE]
+> User-defined tags assigned to your Container Apps environment are replicated to the [Load Balancer](https://azure.microsoft.com/pricing/details/load-balancer/).
```

Comment:
tags are replicated to all resources in the managed resource group in the customer subscription, not just the load balancer

Reply:
@ebencarek Thank you for the feedback. I've updated the document. Could you review it again?